templates/snapcraft.yaml.j2

```diff
@@ -36,11 +44,15 @@ package-repositories:
 architectures:
   - build-on: {{ architecture }}
 
+{% if dev is not defined or not dev | bool -%}
```

Comment:
Shouldn't the "-" be at the beginning of the line for consistency?

Reply:
Having it at the beginning or the end doesn't lead to the same result.
I suppose we could revisit the overall [white-space control](https://jinja.palletsprojects.com/en/3.1.x/templates/#whitespace-control) in a subsequent pr.